src/core/Alert/Alert/Alert.test.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import { Alert } from './Alert';
+import { axeTest } from '../../../utils/test';
+
+describe('children', () => {
+  const alertWithElementChild = (
+    <Alert closeText="Close">
+      <div>Test alert</div>
+    </Alert>
+  );
+
+  it('has the given content', () => {
+    const { container } = render(alertWithElementChild);
+    expect(container.firstChild).toHaveTextContent('Test alert');
+  });
+
+  it('should match snapshot', () => {
+    const { container } = render(alertWithElementChild);
+    expect(container).toMatchSnapshot();
+  });
+});
+
+describe('classnames', () => {
+  const customClassAlert = (
+    <Alert closeText="Close" className="custom-class">
+      Test content
+    </Alert>
+  );
+
+  it('contains base classname', () => {
+    const { container } = render(customClassAlert);
+    expect(container.firstChild).toHaveClass('fi-alert');
+  });
+
+  it('contains custom classname', () => {
+    const { container } = render(customClassAlert);
+    expect(container.firstChild).toHaveClass('custom-class');
+  });
+});
+
+describe('Error view tests', () => {
+  const ErrorAlert = (
+    <Alert
+      status="error"
+      className="alert-test-class"
+      closeText="Close"
+      smallScreen
+    >
+      Testcontent
+    </Alert>
+  );
+  it('should have status and smallScreen prop specific classNames', () => {
+    const { container } = render(ErrorAlert);
+    expect(container.firstChild).toHaveClass('fi-alert--error');
+    expect(container.firstChild).toHaveClass('fi-alert--small-screen');
+  });
```

Comment:
After the removal of the duplicate test of `className` prop, we are only having one test left In this **describe** block.

We could simplify like this and make it more readable when actually testing the status prop that is not yet tested and what is basically left after the removal of duplicate test.

```tsx
  describe('status', () => {
    const ErrorAlert = (
      <Alert status="error" closeText="Close">
        Testcontent
      </Alert>
    );

    test('error status should have correct classname applied', () => {
      const { container } = render(ErrorAlert);
      expect(container.firstChild).toHaveClass('fi-alert--error');
    });

    const WarningAlert = (
      <Alert status="warning" closeText="Close">
        General warning
      </Alert>
    );

    test('warning status should have correct classname applied', () => {
      const { container } = render(WarningAlert);
      expect(container.firstChild).toHaveClass('fi-alert--warning');
    });
  });
```

With the above we have more describing test and it will test `error` and `warning` statuses.

Reply:
And like other props, this describe block of the `status` prop should be inside the existing `props` block.

```tsx
describe('props', () => {
  describe('status', () => {
  // .. stuff here
  });
 // .. other tests for other props here like before
  });
```